File: Linguagem_C/Algoritmos/LogicaParaDesenvolvimentoDeProgramacaoDeComputadores/capitulo_3/exercicio3F.c
#include <stdio.h>

int a, b;

void trocaValores(){

  int x = a;
  a = b;
  b = x;

  return;
}

int main(void){
  
  printf("digite o valor de a:");
  scanf("%d",&a);

  printf("digite o valor de b:");
  scanf("%d",&b);

  trocaValores();

  printf("O valor de a eh: %d\nO valor de b eh: %d\n",a,b);
  return 0;
}

File: Python/Algoritmos/LogicaParaDesenvolvimentoDeComputadores/Capitulo_3/exercicio3F.py
"""
Efetuar a leitura das variaveis A e B e realizar a troca dos valores
"""

def trocaValores(a,b):
    x = a
    a = b
    b = x
    print(f"O valor de A eh:{a}")
    print(f"O valor de B eh:{b}")

a = int(input("Digite o valor de a:"))
b = int(input("Digite o valor de b:"))

trocaValores(a,b)

